twitter.html

```diff
@@ -0,0 +1,333 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Twitter Project</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600&family=Lato:wght@300&family=Roboto:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="twitter1.css" />
+  </head>
+  <body>
+    <div class="container">
+      <!-- LEFT MENU -->
+      <section class="left-menu">
+        <div class="lft-top--el-a">
+          <ion-icon name="logo-twitter" class="twitter"></ion-icon>
+          <div class="left-icons-n-list">
+            <div class="left-m-icons">
+              <ul class="list">
+                <a href="https://twitter.com/">
+                  <li>
+                    <ion-icon name="home" class="house"></ion-icon></li
+                ></a>
+                <a href="#"
+                  ><li>
+                    <p>
+                      <svg
```

Comment:
You could use the svg with an `<img>` tag using the `src` property to point to the SVG file.